ingester/tests/test_storage.py

```diff
@@ -143,3 +143,58 @@ def test_store_conditional_put_latest_multiple_files(dynamodb_latest_table, dyna
     res = dict(dynamodb_latest_table.get_item(what_where_key=query_what_where))
     assert res['metadata']['start'] == file2['metadata']['start']
 
+
+
+def test_insert_ingestion_issue(dynamodb_latest_table, dynamodb_connection):
```

Comment:
Maybe a better name for this test? Is it something like `test_insert_duplicate_writes_once`?

Reply:
Fixed with test_verify_replace_record_same_start. I can change it if we still don't like that.